charts/registry-proxy/templates/registry-proxy-daemon.yaml

```diff
@@ -38,7 +41,15 @@ spec:
           value: $(DEIS_REGISTRY_SERVICE_HOST)
         - name: REGISTRY_PORT
           value: $(DEIS_REGISTRY_SERVICE_PORT)
+        - name: BIND_ADDR
+{{- if (.Values.global.use_cni) }}
+          value: {{ default "127.0.0.1:5555" .Values.global.registry_proxy_bind_addr }}
```

Comment:
oops, looks like we need to add an extra `| quote` to these, or else an error occurs such as:
```
 $ helm install charts/registry-proxy
Error: release right-moth failed: DaemonSet in version "v1beta1" cannot be handled as a DaemonSet: [pos 583]: json: expect char '"' but got char '8'
```

Therefore, can change to:
```
{{ default "127.0.0.1:5555" .Values.global.registry_proxy_bind_addr | quote }}
...
{{ default "80" .Values.global.registry_proxy_bind_addr | quote }}
```